src/content/docs/logs/log-api/introduction-log-api.mdx

```diff
@@ -461,7 +461,8 @@ Attributes may be of any of the following types:
       </td>
 
       <td>
-        (unsupported)
+        Arrays will be _stringified_ and stored as strings. In the case of OpenTelemetry logs,
```

Comment:
I don't think we should be using the verb _stringify_ in our pub docs, I'd rephrase it as: "Arrays will be **converted to text** ..."